flatpak/org.prismlauncher.PrismLauncher.yml

```diff
@@ -1,9 +1,7 @@
 id: org.prismlauncher.PrismLauncher
 runtime: org.kde.Platform
-runtime-version: 6.7
+runtime-version: '6.7'
 sdk: org.kde.Sdk
-sdk-extensions:
-  - org.freedesktop.Sdk.Extension.openjdk17
```

Comment:
Why is this removed? We need Java >= 8 <= 17 to build

Reply:
As my comments above expained, @TheKodeToad made this mistake on Flathub and will need to revert the change over there too. Because it was definitely needed by the build process. I was already in the process of reverting this change here. Hold a moment. :)